src/amo/api/hero.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+import { callApi } from 'core/api';
+import type { ApiState } from 'core/reducers/api';
+import type { ExternalHeroShelvesType } from 'amo/reducers/home';
+
+export type GetHeroShelvesParams = {| api: ApiState |};
+
+export const getHeroShelves = ({
+  api,
+}: GetHeroShelvesParams): Promise<ExternalHeroShelvesType> => {
+  return callApi({
+    apiState: api,
+    auth: true,
+    endpoint: 'hero/',
```

Comment:
not sure we need the trailing slash here

Reply:
Looks like you are correct. I looked at another api implementation for inspiration, and it looks like about half of the code we have for api endpoints use a trailing slash and half do not. 😞

Reply:
The underlying code handles all cases anyway, so it does not really matter much.